File: C++/Easy/0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& arr) {
            int n = arr.size();
            int left = 0;
            int right = 1;
            int index=0;
            while(left<=right && right<n){
                if(arr[left]!=arr[right]){
                    left++;
                    arr[left]=arr[right];
                }
                right++;
          }
        return left+1;
    }
};

File: Python3/Easy/0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        L=[]
        cnt=0
        for i in nums:
            
            if i not in L:
                L.append(i)
                cnt+=1

        for j in range (0, len(L)):
            nums[j]=L[j]


        return cnt

        